test/index.html

```diff
@@ -11,16 +11,16 @@
 <body>
   <script>
     WCT.loadSuites([
-      'vaadin-combo-box-properties.html',
-      'toggling-dropdown.html',
-      'overlay-position.html',
-      'selecting-items.html',
-      'form-input.html',
-      'filtering.html',
-      'keyboard.html',
-      'scrolling.html',
-      'adaptive-overlay.html',
-      'aria.html'
+      // 'vaadin-combo-box-properties.html',
```

Comment:
You probably already noticed, but this seems like a mistake that these are commented out.